truenas/webui
# NAS-117581 / 22.02.4 / NAS-117581: Fix Launch Docker Image button bugs

**Testing**

1. Go to **Apps** page  > **Launch Docker Image**
2. See bug description in the ticket
